`TestDevPortForwardDeletePod` failing integration test against Minikube on Darwin

<details><summary>Error log</summary>

```
=== Failed Tests ===
/integration/TestDevPortForwardDeletePod
time="2020-08-28T10:21:47Z" level=info msg="Namespace: skaffoldqcht4"
time="2020-08-28T10:21:47Z" level=info msg="[skaffold dev --namespace skaffoldqcht4 --default-repo gcr.io/k8s-skaffold --port-forward --rpc-port 63274]"
    rpc_test.go:377: waiting for connection...
    rpc_test.go:377: waiting for connection...
    dev_test.go:267: event received timestamp:<seconds:1598610108 nanos:533826000 > event:<metaEvent:<entry:"Starting Skaffold: &{Version:67e81bc ConfigVersion:skaffold/v2beta7 GitVersion: GitCommit:67e81bce7eba453dcca2b57a92e8157dbca3ecc3 GitTreeState:clean BuildDate:2020-08-28T06:00:45Z GoVersion:go1.14.2 Compiler:gc Platform:darwin/amd64}" metadata:<build:<numberOfArtifacts:2 builders:<type:DOCKER count:2 > type:LOCAL > deploy:<deployers:<type:KUBECTL count:1 > cluster:MINIKUBE > > > > 
    dev_test.go:267: event received timestamp:<seconds:1598610108 nanos:533920000 > event:<devLoopEvent:<status:"In Progress" > > entry:"Update initiated due to file change" 
    dev_test.go:267: event received timestamp:<seconds:1598610110 nanos:375950000 > event:<buildEvent:<artifact:"leeroy-web" status:"In Progress" > > entry:"Build started for artifact leeroy-web" 
WARNING: Could not open the configuration file: [/Users/runner/.config/gcloud/configurations/config_default].
ERROR: (gcloud.auth.docker-helper) You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials, or if you have already logged in with a
different account:

  $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
    dev_test.go:253: timed out waiting for port forwarding event
time="2020-08-28T10:22:49Z" level=info msg="Ran in 1m2.012523065s"
--- FAIL: TestDevPortForwardDeletePod (62.04s)
```
</details>

Failed run: [here](https://github.com/GoogleContainerTools/skaffold/runs/1040946086)

--- a/integration/skaffold/helper.go
+++ b/integration/skaffold/helper.go
@@ -169,7 +169,7 @@ func (b *RunBuilder) WithProfiles(profiles []string) *RunBuilder {
 
 // RunBackground runs the skaffold command in the background.  The Skaffold output
 // is accumulated and logged on test failure.
-func (b *RunBuilder) RunBackground(t *testing.T) io.Reader {
+func (b *RunBuilder) RunBackground(t *testing.T) {
 	t.Helper()
 	out := bytes.Buffer{}
 	b.runForked(t, &out)
@@ -179,7 +179,6 @@ func (b *RunBuilder) RunBackground(t *testing.T) io.Reader {
 			t.Log("Skaffold log:\n", strings.ReplaceAll(out.String(), "\n", "\n> "))
 		}
 	})
-	return &out
 }
 
 // RunLive runs the skaffold command in the background with live output.

